src/styles/layout/_header.scss

```diff
@@ -0,0 +1,44 @@
+.header-container {
+  font-family: "Quicksand", sans-serif;
+  width: 100%;
+  font-weight: 500;
+  height: 111px;
+  position: absolute;
+  z-index: 99;
+  background-color: transparent;
+
+  .container {
+    height: 100%;
+  }
+
+  .row {
+    height: 100%;
+  }
+
+  .nav {
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
```

Comment:
tại sao set là 100%

Reply:
- Để nav có thể ăn được toàn bộ chiều cao của header, nếu không nó chỉ ăn đủ chiều cao của logo và text nên sẽ không căn giữa theo chiều dọc của toàn bộ phần header
![309774469_2398663580302310_3941906714769192089_n](https://user-images.githubusercontent.com/99970939/202599853-a8df6e50-dafa-4679-b9b8-1237404297bc.png)